templates/options.conf.erb

```diff
@@ -4,6 +4,11 @@ forwarders { <%= scope.lookupvar('::dns::forwarders').join("; ") %>; };
 <% end -%>
 
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
+
+<% unless scope.lookupvar('::dns::allow_recursion').empty? -%>
```

Comment:
I've noticed that [since Bind 9.4.1-P1 the default is safer](https://kb.isc.org/article/AA-00269/0/What-has-changed-in-the-behavior-of-allow-recursion-and-allow-query-cache.html) so now I think the current template is good.